main.cc

```diff
@@ -1690,7 +1689,7 @@ To start the scylla server proper, simply invoke as: scylla server (or just scyl
             }).get();
 
             with_scheduling_group(maintenance_scheduling_group, [&] {
-                return ss.local().join_cluster(sys_dist_ks, proxy);
+                return ss.local().join_cluster(sys_dist_ks, proxy, gossiper, true);
```

Comment:
Maybe you could wrap `true` in a variable so we know what it stands for.

Reply:
Alternatively, you can use `seastar::bool_class`:

```c++
using start_hints_manager = seastar::bool_class<class start_hints_manager_tag>;

// ...

return ss.local().join_cluster(sys_dist_ks, proxy, gossiper, start_hints_manager::yes);
```